spec/datarockets/style/cop/style/nested_interpolation_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe Datarockets::Style::Cop::Style::NestedInterpolation do
+  subject(:cop) { described_class.new }
+
+  it "accepts single interpolation" do
+    expect_no_offenses('"dear #{user.name}"')
+  end
+
+  it "accepts two single interpolation" do
+    expect_no_offenses('"Hello #{user1.name} and #{user2.name}"')
+  end
+
+  it 'accepts interpolation with or logic' do
+    expect_no_offenses('"\n#{(lines.join("\n").split(node.source).first || \'\')}"')
+  end
+
+  it 'accepts interpolation into business block' do
+    expect_no_offenses('"#{assignment.join("\n#{indentation(node)}")}"')
```

Comment:
How do you think should we ban that construction? Or we can accept it?

Reply:
@roman-dubrovsky , I think it is redundant and should be banned.